src/com/facebook/buck/swift/SwiftDescriptions.java

```diff
@@ -70,4 +71,15 @@ private SwiftDescriptions() {
     output.bridgingHeader = args.bridgingHeader;
   }
 
+  static boolean isSharedRequested(Linker.LinkableDepType type) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1457567/18166470/38e6efe8-7000-11e6-822b-070ae5c75af3.png)

Indeed, we use the comparison everywhere, I guess it is safe to use it again :trollface: